lib/spack/docs/module_file_support.rst

```diff
@@ -307,7 +309,7 @@ under the Spack root:
   +========================================+====================================+========================+
   |  **Dotkit Module Files**               |   share/spack/dotkit               |  DotKit                |
   +----------------------------------------+------------------------------------+------------------------+
```

Comment:
This table needs a heading.  As is, it looks like the dotkit stuff is the heading.
Also... I would recommend NOT including dotkit in the docs. LLNL is the only site using dotkit, and they are switching to Lmod.